app/scripts/modules/titus/migration/titusMigrationConfig.component.ts

```diff
@@ -7,6 +8,11 @@ import './titusMigrationConfig.component.less';
 
 export interface ITitusMigrationViewState extends IViewState {
   accountsLoaded: boolean;
+  isDirty: boolean;
+  originalConfig: ITitusMigrationConfig;
+  originalStringVal: string;
+  saveError: boolean;
+  saving: boolean;
```

Comment:
This seems like a bigger problem. `isDirty`, `originalStringVal`, `saveError`, and `saving` are all declared in `IViewState`, so we shouldn't have to re-declare them here, unless extending an interface is suddenly not supported in TS.